### PR TITLE
Add interface includes for PDAL library target.

### PR DESCRIPTION
**Pull request by abellgithub**

### CODE REVIEW

**Comment by abellgithub** [comment]
Close #2262

File: CMakeLists.txt [hunk 1]
```diff
@@ -378,6 +378,9 @@ install(
     DESTINATION
         "${PDAL_LIB_INSTALL_DIR}/cmake/PDAL")
 include(${PDAL_CMAKE_DIR}/config.cmake)
+target_include_directories(${PDAL_BASE_LIB_NAME}
+    INTERFACE
+        $<INSTALL_INTERFACE:${CMAKE_INSTALL_PREFIX}/include>)
```
Comment (hobu) [inline CMakeLists.txt]: Do we need the same for pdal_util as well?

Comment (abellgithub) [reply CMakeLists.txt]: Not need.  The only reason is if you're using the utilities without the main PDAL lib.  But I can add it.

Comment (hobu) [reply CMakeLists.txt]: we should do so in case someone follows a pattern from a PDAL filter/reader in their own plugin or something.

**Pull request merged**